.gitignore

```diff
@@ -2,10 +2,11 @@
 __pycache__
 *.egg-info
 .pytest_cache
+.ruff_cache
```

Comment:
Interesting... it doesn't show up in this PR? I think it gets created by our pre-commit hook, so it may have been created locally when you last made changes